### ISSUE TITLE
Change to deno.land/x subfolder

### ISSUE THREAD

**Issue by yandeu** [garronej/denoify#10]
Currently, I'm using v0.3.8. 

In >0.4.0, `mod.ts` will be placed inside `deno_lib`.

I guess I only have to append `?subdir=deno_lib%2F` to the github hook in order to publish `deno_lib/` to [deno.land/x](https://deno.land/x)?

So instead of `https://api.deno.land/webhook/gh/nano_jsx`, it will be `https://api.deno.land/webhook/gh/nano_jsx?subdir=deno_lib%2F`?

---

Does `deno_lib/` need to contain a `README.md` file? Will the `README.md` file inside `deno_lib/` be visible on [deno.land/x](https://deno.land/x) or the main readme file of the repository?

**Comment by garronej** [comment]
Yes, you only need to update the webhook URL, for you il would be: `https://api.deno.land/webhook/gh/nano_jsx?subdir=deno_lib%252F`, you can also remove `mod.ts` from `tsconfig.json` excludes.

> Does `deno_lib/` need to contain a `README.md` file? Will the `README.md` file inside `deno_lib/` be visible on [deno.land/x](https://deno.land/x) or the main readme file of the repository?

Yes, `deno_lib/` needs to contain the `READM.md`, the root `README.md` is automatically copied in by Denoify.

Regards
